engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -167,9 +167,11 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
     //Success
     // Job
     case Event(r: JobSucceededResponse, stateData) =>
+      workflowLogger.info(s"Job succeeded: '${r.jobKey.call.fullyQualifiedName}' (scatter index: ${r.jobKey.index}, attempt ${r.jobKey.attempt})")
```

Comment:
These are potentially controversial (especially the expression one), but were helpful for peace of mind while debugging